applications/build-modular-monolith-with-platformatic/services/media-service/plugin.js

```diff
@@ -0,0 +1,54 @@
+/// <reference path="../books-service/clients/persons-service/persons_service.d.ts" />
+// TODO: The client will be moved if I can get things working in the Composer `onRoute` hook.
+
+'use strict'
+
+/** @param {import('fastify').FastifyInstance} app */
+module.exports = async function (app) {
+
+  // /books/
+  // /books/{id}
+
+  app.log.info('Media service plugin loaded')
+
+  // Modify route: GET /books
+  app.platformatic.addComposerOnRouteHook('/books/', ['GET'], routeOptions => {
+    const responseSchema = routeOptions.schema.response[200].items
+    responseSchema.properties.authorName = {
+      type: 'string'
+    }
+    responseSchema.required.push('authorName')
+
+    async function onComposerResponse (request, reply, body) {
+      // This message is never logged
+      app.log.info('CALLED: onComposerResponse')
+
+      let payload
+
+      try {
+        // This causes the parent Runtime service to crash out without an error
+        payload = await body.json()
```

Comment:
I'm not sure why, but this line causes the parent Runtime service to crash out without an error (I'm testing this with the latest version of the `platformatic` repo's `main` branch).

To reproduce:

- Clone this repo
- Check out this branch: `build-modular-monolith-with-platformatic`
- Run these commands:

```bash
cd applications/build-modular-monolith-with-platformatic/

# Change this to the path to your local platformatic repo (make sure it's on `main` and up-to-date)
npm install ../path/to/platformatic/packages/cli

cd services/books-service/
npm install
cp .env.sample .env
npx platformatic db migrations apply
npx platformatic db seed seed.js

cd ../movies-service/
npm install
cp .env.sample .env
npx platformatic db migrations apply
npx platformatic db seed seed.js

cd ../persons-service/
npm install
npx platformatic db migrations apply
npx platformatic db seed seed.js

# Go back to the project root
cd ../..

# Start the Runtime app
npm start
```

Then make a request with cURL to the Media service (Composer), which is exposed by the Runtime app:

```bash
curl localhost:3045/books/
```

You should see that the Runtime app crashes without any error and cURL will output `curl: (52) Empty reply from server`.